modules/core/src/main/scala/shop/AppResources.scala

```diff
@@ -23,7 +25,7 @@ final case class AppResources[F[_]](
 
 object AppResources {
 
-  def make[F[_]: Concurrent: ContextShift: Logger: Timer](
+  def make[F[_]: Async: Console: Logger: Network](
```

Comment:
Yeah I left them intentionally to be explicit about the requirements. `Async` is only needed for `redis4cats` here, maybe I can extract this out to avoid this hard constraint.

Reply:
Should be able to remove it after https://github.com/profunktor/redis4cats/pull/516 gets in :)